generic/tests/iozone_windows.py

```diff
@@ -63,9 +63,14 @@ def get_driver():
     session = vm.wait_for_login(timeout=timeout)
     driver_name = get_driver()
     if driver_name:
-        session = utils_test.qemu.windrv_check_running_verifier(session, vm,
-                                                                test, driver_name,
+        driver_id_pattern = params["driver_id_pattern"]
+        devcon_path = utils_misc.set_winutils_letter(session, params["devcon_path"])
```

Comment:
`devcon_path = "WIN_UTILS:\devcon\x86\"` probably needs to added in upstream as well, or any one who run the script will be failed.

Reply:
Updated it in upstream guest-os file in https://github.com/avocado-framework/avocado-vt/pull/1892
Thank you for the advice.